qiskit_nature/operators/second_quantization/__init__.py

```diff
@@ -34,11 +34,13 @@
 from .second_quantized_op import SecondQuantizedOp
 from .spin_op import SpinOp
 from .vibrational_op import VibrationalOp
+from .sparse_label_op import SparseLabelOp
 
 __all__ = [
     "FermionicOp",
     "QuadraticHamiltonian",
     "SecondQuantizedOp",
     "SpinOp",
     "VibrationalOp",
+    "SparseLabelOp",
```

Comment:
This should be added to the dosctring list above to I think - this way sphinx will also include it so it will check the docstring correctness as well.